force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +467,12 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if (person1.age > person2.age) {
+            return Person1;
+        }
+        else if (person2.age > person1.age) {
+            return person2; 
+        }
+        return person1;
```

Comment:
Good work.  Could you be more concise? See if you can do it with out the else if.